Tests/MongoSwiftTests/CrudTests.swift

```diff
@@ -401,7 +405,12 @@ private class DistinctTest: CrudTest {
         let resultDoc: Document = ["result": try coll.distinct(fieldName: fieldName,
                                                                filter: filter ?? [:],
                                                                options: options)]
-        let expectedDoc: Document = ["result": self.result]
+        let expectedDoc: Document
+        if let result = self.result {
+            expectedDoc = ["result": result]
```

Comment:
The point I raised in https://github.com/mongodb/mongo-swift-driver/pull/145/files#r236746341 may apply here as well. I think `expect(resultDoc).to(equal(expectedDoc))` should be limited to the condition where `result` is non-nil.

Reply:
Just read this comment after posting my question in response to the point raised in that link. It seems like it's actually correct for us to just avoid the assertion if `self.result` is `nil`?

Reply:
Done!